### ISSUE TITLE
Questions about Loss in paper

### ISSUE THREAD

**Issue by WentingXu3o3** [facebookresearch/locate-3d#20]
Hi authors,

Thank you for this amazing work and for maintaining the code so well. I was going through Appendix C.1 on the loss function and found a point that left me a bit confused. Specifically, the formulation shown as "L = λ<sub>dice</sub>L<sub>dice</sub> + λ<sub>ce</sub>L<sub>ce</sub> + λ<sub>box</sub>L<sub>box</sub> + λ<sub>giou</sub>L<sub>giou</sub> + λ<sub>align</sub>L<sub>align</sub>" does not mention "λ<sub>mask</sub> = 6.0 (Mask cross entropy weight)"  which is described later. Additionally,  "λ<sub>align</sub>" is included in the equation but not defined or explained in the accompanying text.
I’ve attached a screenshot of the relevant section for reference. Could you please clarify how λ<sub>mask</sub> and λ<sub>align</sub> are used in the final loss?

Thanks again for your great work!

<img width="807" alt="Image" src="https://github.com/user-attachments/assets/83c8d50e-7ca4-4c11-a6f5-59c770139c76" />
<img width="824" alt="Image" src="https://github.com/user-attachments/assets/55c9a0a3-873f-4f87-9ca9-8f5ff780ea4b" />

**Comment by SergioArnaud** [comment]
Thanks for catching this error in the appendix; the correct statement would be, λalign is incorrect:

```
L =  λceLce + λmaskLmask + λdiceLdice +  λboxLbox + λgiouLgiou
```

With the weights as described in the second image.

The main writing says:
```
Specifically, LOCATE 3D optimizes a composite loss function, which includes: (1) a mask loss, combining Dice and cross-entropy loss terms (Cheng et al., 2021); (2) a bounding box loss, composed of L1 distance and generalized IoU (Carion et al., 2020) terms; and (3) a text alignment focal loss with alpha balancing. Following (Carion et al., 2020), we define a matching cost and use Hungarian Matching to assign object query predictions to ground truth objects.
```

This is correct, and the weights of each one of the 5 components from the writting are the ones from the second image you attached. 

LMK if there are further questions.

**Comment by WentingXu3o3** [comment]
Hi @SergioArnaud, thank you for the answer. 
However, would there be no "(3) a text alignment focal loss with alpha balancing" in the corrected loss function?
```
L =  λceLce + λmaskLmask + λdiceLdice +  λboxLbox + λgiouLgiou
```

**Comment by SergioArnaud** [comment]
That would be λce

**Comment by WentingXu3o3** [comment]
> That would be λce
Thank you so much. 🌹

**Comment by SergioArnaud** [comment]
You welcome, LMK if there are any further questions. We understand this writting about the loss was a bit ambiguous

**Comment by WentingXu3o3** [comment]
> You welcome, LMK if there are any further questions. We understand this writting about the loss was a bit ambiguous

Sure. 🫰
